apps/nextjs/src/lib/feature-flags/bootstrap.ts

```diff
@@ -32,15 +32,21 @@ function getDistinctIdFromCookie(headers: ReadonlyHeaders) {
 }
 
 export async function getBootstrappedFeatures(headers: ReadonlyHeaders) {
-  const { userId } = auth();
+  const { userId, sessionClaims } = auth();
 
   const distinctId = userId ?? getDistinctIdFromCookie(headers) ?? "0";
   log.info("Evaluating feature flags for", distinctId);
+
+  const personProperties = sessionClaims?.labs?.featureFlagGroup
```

Comment:
Is labs already decided? I believe we're using "aila" as the name for everything now

Reply:
Yeah, we already have a number of keys under labs. It comes from when the teacher squad started their proof of concept for Clerk. They have `owa` and we have `labs`. There's no reason why we can't change it, but we'd need some code to read both for a time